docs/LwM2M_Client/OpenThread/OTBR_with_svetovid.md

```diff
@@ -0,0 +1,183 @@
+# OpenThread Border Router configurable through LwM2M server
+
+In this tutorial, we will configure OpenThread Border Router with [**Svetovid**](https://github.com/AVSystem/Svetovid-raspberry-client) which is an LwM2M Client for Linux-based devices based on [**Anjay**](https://anjay.io).
+With the help of **Svetovid**, we will be able to manage the Border Router remotely, through the LwM2M server. Within this guide, we will use the {{ coiote_long_name }} as the server.
+
+## Prerequisites
+
+- The [**nRF52840 DK**](https://www.nordicsemi.com/Products/Development-hardware/nrf52840-dk) or the [**nRF52840 Dongle**](https://www.nordicsemi.com/Products/Development-hardware/nrf52840-dongle).
+- A Linux-based machine (in this tutorial we will use Raspberry Pi 3 model B with Raspbian) with `AMD64`, `ARMv7` or `ARM64` architecture.
```

Comment:
Wait if we are running Svetovid inside a docker container then is the architecture information relevant?

Reply:
Yep, there may also be a different version depending on the OS: https://docs.docker.com/build/building/multi-platform/